src/Todoist.Net/Models/DueDate.cs

```diff
@@ -80,7 +80,8 @@ internal DueDate()
         /// <remarks>
         /// Format date according this rules https://developer.todoist.com/sync/v9/?shell#due-dates
         /// </remarks>
-        [JsonProperty("date")]
+        [JsonInclude]
```

Comment:
Please leave a comment why do we need the attribute.

Reply:
For our bad luck, developers of `System.Text.Json` have decided not to serialize\deserialize `internal` members, I am not sure exactly why 🤔
They don't even provide a global option to include all internal properties by default, and we have to either use `JsonIncludeAttribute` for every property or write an entire `JsonConverter`.

For public properties with an `internal` setter, I did actually write a `JsonConverter` to include them because they were too many. Still, this `StringDate` property is the only non-public property we want to include, so I just used the attribute.